interface/src/app/tribe_auth/tribe_user.js

```diff
@@ -0,0 +1,61 @@
+angular.module('adage.tribe_auth.user', [
+    'adage.tribe_auth.resource',
+])
+
+.factory( 'UserFactory', function( User ) {
```

Comment:
Good habit. Thank you to @dongbohu for tracking this down while reviewing one of my PRs and giving us a name for it.